locale/ru/about/governance.md

```diff
@@ -7,30 +7,18 @@ layout: about.hbs
 
 ## Процесс поиска консенсуса
 
-Проект Node.js следует модели [поиска консенсуса][] для принятия решений.
+Проект Node.js следует модели [поиска консенсуса](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) для принятия решений.
 
 ## Коллабораторы
 
-Репозиторий [nodejs/node][] на GitHub поддерживается и развивается за счет Коллабораторов,
-которые добавляются Техническим руководящим комитетом ([TSC][]) на постоянной основе.
+Репозиторий [nodejs/node](https://github.com/nodejs/node) на GitHub поддерживается и развивается за счет Коллабораторов, которые добавляются Техническим руководящим комитетом ([TSC](https://github.com/nodejs/TSC)) на постоянной основе.
 
-Люди, которые вносят значительный и ценный вклад, становятся Коллабораторами и получают постоянный
-доступ к проекту. Эти лица идентифицируются TSC, и их назначение обсуждается с существующими сотрудниками.
+Люди, которые вносят значительный и ценный вклад, становятся Коллабораторами и получают постоянный доступ к проекту. Эти лица идентифицируются TSC, и их назначение обсуждается с существующими сотрудниками.
 
-Текущий список Коллабораторов можно найти в [README.md][] проекта.
+Текущий список Коллабораторов можно найти в [README.md](https://github.com/nodejs/node/blob/master/README.md#current-project-team-members) проекта.
 
-Руководство для Коллабораторов находится по адресу [collaborator-guide.md][].
+Руководство для Коллабораторов находится по адресу [collaborator-guide.md](https://github.com/nodejs/node/blob/master/doc/guides/collaborator-guide.md).
 
 ## Комитеты высшего уровня
 
-Проект управляется совместно [Техническим руководящим комитетом (TSC)][], который отвечает за
-руководство проектом на высоком уровне, и [Комитетом сообщества (CommComm)][], который отвечает
-за руководство и расширение Node.js сообщества.
-
-[collaborator-guide.md]: https://github.com/nodejs/node/blob/master/doc/guides/collaborator-guide.md
```

Comment:
@nodejs/nodejs-ru please do not reformat the anchor links. this will cause issues going forward syncing with the English test